### ISSUE TITLE
Wrong NodeJS download URL for ARMv6

### ISSUE THREAD

**Issue by Alwaysin** [wallabag/wallabag#2461]
### Issue details
````
Generating optimized autoload files
Installing NodeJS v6.8.1
  Downloading from https://nodejs.org/dist/v6.8.1/node-v6.8.1-armv6l.tar.gz
    Downloading: Failed


  [Composer\Downloader\TransportException]
  The "https://nodejs.org/dist/v6.8.1/node-v6.8.1-armv6l.tar.gz" file could not be downloaded (HTTP/1.1 404 Not Fo
  und)
```

Right URL has an added "linux-" like that: https://nodejs.org/dist/v6.8.1/node-v6.8.1-linux-armv6l.tar.gz

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.1.1
* How did you install wallabag? Via `git clone` or by downloading the package? git clone
* Last wallabag version that did not exhibit the issue (if applicable): I don't know
* php version: 7.0.12
* OS: Arch Linux on ARMv6
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MariaDB, but not able to do that far :)

### Steps to reproduce/test case
```
 git clone https://github.com/wallabag/wallabag.git
 cd wallabag/
 make install
```

**Comment by Alwaysin** [comment]
Also, I have installed NodeJS directly onto the server to try to bypass the issue but it still wants to download it... Is that normal?
```
# node -v
v6.8.1
```

**Comment by j0k3r** [comment]
This might be fixed with the upcoming 2.1.2 version because we removed the deps of Node.js.
